reana_commons/config.py

```diff
@@ -117,16 +117,39 @@ def load(cls, kind):
 By default runtime pods will run in the same namespace as the infrastructure pods.
 """
 
-REANA_RUNTIME_KUBERNETES_NODE_LABEL = (
-    {
-        os.getenv("REANA_RUNTIME_KUBERNETES_NODE_LABEL")
-        .split("=")[0]: os.getenv("REANA_RUNTIME_KUBERNETES_NODE_LABEL")
-        .split("=")[1]
-    }
-    if os.getenv("REANA_RUNTIME_KUBERNETES_NODE_LABEL")
-    else {}
+
+def kubernetes_node_label_to_dict(node_label):
+    """Load Kubernetes node label to Python dict."""
+    if node_label:
+        label_name, value = node_label.split("=")
+        return {label_name: value}
+
+    return {}
+
+
+REANA_RUNTIME_BATCH_KUBERNETES_NODE_LABEL = kubernetes_node_label_to_dict(
+    os.getenv("REANA_RUNTIME_BATCH_KUBERNETES_NODE_LABEL")
+)
+"""Kubernetes label (with format ``lable_name=lable_value``) which identifies the nodes where the runtime batch workflows should run.
```

Comment:
```suggestion
"""Kubernetes label (with format ``label_name=label_value``) which identifies the nodes where the runtime batch workflows should run.
```

Same happens twice below (jobs and sessions).